docs/contributing.rst

```diff
@@ -4,150 +4,122 @@
 Contributing
 ************
 
-If you are thinking about making Mopidy better, or you just want to hack on it,
-that’s great. Here are some tips to get you started.
+If you want to contribute to Mopidy, here are some tips to get you started.
 
 
-Getting started
-===============
+.. _asking-questions:
 
-#. Make sure you have a `GitHub account <https://github.com/signup/free>`_.
+Asking questions
+================
 
-#. If a ticket does not already exist `submit a ticket
-   <https://github.com/mopidy/mopidy/issues/new>`_ for your issue.
-   Make sure to clearly describe the issue, and if it is a bug: include steps
-   to reproduce.
+Please use one of these channels for requesting help with Mopidy and its
```

Comment:
> Please get in touch with us in one of these ways when requesting help with Mopidy and its extensions:

Perhaps, feels less technical/formal that the implied communication channels IMO.